src/analysis/retail/shaman/restoration/modules/talents/AncestralVigor.tsx

```diff
@@ -127,11 +138,65 @@ class AncestralVigor extends Analyzer {
         />
       );
     } else {
+      // Because we now have 2 buffs giving +10% max hp, we have to consider that the bonus to account for is a +20% max hp,
+      // and filter the events according to the active buffs at the time of the DamageEvent
+      this.lifeSavingEvents.map((event, index) => {
+        const combatant = this.combatants.getEntity(event);
+        if (!combatant) {
+          return null;
+        }
+
+        // Check for duplicates, based on timestamp
+        if (this.filteredLifeSavingEvents.find((e) => e.timestamp === event.timestamp)) {
+          return null;
+        }
+
+        // Check presence of buffs in the timeframe
+        if (
+          !combatant.hasBuff(SPELLS.DOWNPOUR_HEAL.id, event.timestamp, 100, 50) &&
+          !combatant.hasBuff(SPELLS.ANCESTRAL_VIGOR.id, event.timestamp, 100, 50)
+        ) {
+          return null;
+        }
+
+        // Compute the current max HP bonus from our buffs
+        const currentDownpourIncrease = combatant.hasBuff(
+          SPELLS.DOWNPOUR_HEAL.id,
+          event.timestamp,
+          100,
+          50,
+        )
+          ? DOWNPOUR_INCREASED_MAX_HEALTH
+          : 0;
+        const currentAncestralVigorIncrease = combatant.hasBuff(
+          SPELLS.ANCESTRAL_VIGOR.id,
+          event.timestamp,
+          100,
+          50,
+        )
+          ? ANCESTRAL_VIGOR_INCREASED_MAX_HEALTH
+          : 0;
+
+        // compute health ratio relevant for the statistic, with the current active buffs
+        const currentBonusHealthRatio =
+          1 - 1 / (1 + currentAncestralVigorIncrease + currentDownpourIncrease);
```

Comment:
Buffs that provide a percentage increase multiplicative, not additive. For example, with a base HP of `4016122` having both buffs will give a maximum hp of `4859508` not `4819346` (difference of `40161`).

`(1 + 0.1) * (1 + 0.1) = 1.21`, vs `1 + (0.1 + 0.1) = 1.20` which is exactly the difference (1% max hp)

Reply:
Good to know, I was so convinced in stacked additively that I didn't even test it. it is fixed now.